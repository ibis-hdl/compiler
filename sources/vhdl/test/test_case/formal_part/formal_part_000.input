-- formal_part is simply formal_designator
formal_designator
