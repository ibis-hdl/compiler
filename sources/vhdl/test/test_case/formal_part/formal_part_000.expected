(formal_part
  (formal_designator
    (name<v>
      (identifier
        formal_designator
      )
    )
  )
)
